.github/workflows/continuous-integration-workflow.yml

```diff
@@ -37,6 +37,21 @@ jobs:
       # see https://github.com/actions/virtual-environments/issues/3812
       options: --security-opt seccomp=unconfined
     steps:
+      - name: Checkout desul
```

Comment:
If we want to test both external and internal, this would need to be protected with something similar to `maybe_disable_death_tests` or `maybe_use_external_gtest` steps below.

Reply:
The remainder of our CI, especially everything using `jenkins`, uses the internal one. I would prefer not to add another option but I could do it if people prefer.